code/common/finspace.q

```diff
@@ -54,9 +54,8 @@ notifyhdb:{[cluster;changeset]
       
       // Ensuring that the changeset has successfully created before doing the HDB reload
       current:.finspace.checkstatus[(`.aws.get_changeset;.finspace.database;changeset[`id]);("COMPLETED";"FAILED");00:01;0wu];
-      .lg.o[`notifyhdb;("notifying ",string[cluster]," to repoint to changeset ",changeset[`id])];
-      awsdb:.aws.db[.finspace.database;changeset[`id];.finspace.cache;.finspace.dataview];
-      .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[awsdb];.aws.sdep[.finspace.hdbreloadmode]]
+      .lg.o[`notifyhdb;("changeset ",changeset[`id]," ready, bringing up new hdb cluster")];
+      .lg.o[`notifyhdb;"new changeset ready. create new hdb"];
```

Comment:
do we need both of these? seems a bit redundant?

Reply:
Yes, I can see if being a bit redundant. Right now, the contents of the second log are the what is monitored on AWS in order to trigger the creation of the new HDB cluster (the former are just more informative). I suppose I could point CloudWatch to monitor instances of the latter part of the first log message (ie. "ready, bring up new hdb cluster") instead

Reply:
Resolved now